core/lib/include/irq.h

```diff
@@ -98,5 +98,5 @@ MAYBE_INLINE bool irq_is_in(void);
 }
 #endif
 
-#endif /* IRQ_H */
 /** @} */
+#endif /* IRQ_H */
```

Comment:
```suggestion
#endif /* IRQ_H */
```

Would you mind adding a final `\n`?

(Btw: What editor are you using? Many editors/IDEs these days support providing some config file to a project to configure a project specific coding style such indent width, use of tabs/spaces for indent, and whether to add a final newline. It has become a bit of a pet project of mine to add these to our repo, so that more and more users have a good out-of-the box experience when editing code in RIOT.)

Reply:
Can do. I'm using VS Code

Reply:
Hmm, for VS code we actually have the setting already in the repo. Looks like there is an issue with that setting :/

https://github.com/RIOT-OS/RIOT/blob/a605d05eca857a3660b4a03b133fa41ed98d4d1c/.vscode/settings.json#L7

Reply:
That sounds plausible. Good to hear that we probably have not misconfigured the VS Code config then :)